### ISSUE TITLE
d.setMdmNumPreamble

### ISSUE THREAD

**Issue by chrisVat** [atlas0fd00m/rfcat#97]
Hello, I am having an issue with d.setMdmNumPreamble. If I change the value more than once, I am no longer able to send/receive signals until I disconnect and reconnect the Yard Stick One. Any help would be greatly appreciated.

Edit: 
Specifically, the issue arises when changing from a larger Preamble Num to a smaller one. 
	d.setMdmNumPreamble(200) 
	d.setMdmNumPreamble(80) 

This results in my device which is expecting a preamble of size 80 not detecting anything, but if I comment out the first line, remove the Yard Stick One, and then plug it back in and re-run my code, it will detect the signals I am transmitting.

Edit: 
I see now in the documentation that the options for NUM_PREAMBLE are: [2, 3, 4, 6, 8, 12, 16, 24 ]. Would it be possible to get a preamble of 20?

**Comment by Loumiakas** [comment]
This setting seems to be working for me - I can set preamble multiple times without any problem, and this setting will propagate to the modem. You might want to update boot loader of your device and see if that fixes your problem.

I am not too familiar with the preamble setting, but looking at the source code and cc1111 documentation, I tend to agree that that to set minimum preamble bytes to be transmitted must be one of [2, 3, 4, 6, 8, 12, 16, 24 ]. 
![image](https://user-images.githubusercontent.com/44226245/104076593-6dcb6c80-520e-11eb-8ba8-5a91b91792ce.png)

In regards to the actual setting, I observed one peculiar thing that I was hoping someone with more knowledge/experience could explain:
 
When setting preamble value through setMdmNumPreamble function, and printing out current modem configuration, I observed said value and it does not correlate to the set value (It might be correct, I just can't seem to figure out how it's set).  I thought that if I pass an argument of say '3', minimum TX preamble will be 3, but in reality it will be set to 2... Practically, none of the values passed into said function will set minimum TX preamble to the value I have passed in. For example:

```d.setMdmNumPreamble(2)``` will produce ```Min TX Preamble:     2 bytes```
```d.setMdmNumPreamble(3)``` will produce ```Min TX Preamble:     2 bytes```
```d.setMdmNumPreamble(4)``` will produce ```Min TX Preamble:     2 bytes```
```d.setMdmNumPreamble(6)``` will produce ```Min TX Preamble:     2 bytes```
```d.setMdmNumPreamble(8)``` will produce ```Min TX Preamble:     2 bytes```
```d.setMdmNumPreamble(12)``` will produce ```Min TX Preamble:     2 bytes```
```d.setMdmNumPreamble(16)``` will produce ```Min TX Preamble:     3 bytes```
```d.setMdmNumPreamble(24)``` will produce ```Min TX Preamble:     3 bytes```

Is the argument that we are passing into function referring to number of bits? If that's the case, is the byte size 8 bits? And if so, why does ```d.setMdmNumPreamble(16)``` produce ```Min TX Preamble:     3 bytes```?
